boards/arm/nrf52840_papyr/nrf52840_papyr.dts

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2019 Ajay Meena <intajayk@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+
+/ {
+	model = "Papyr";
+	compatible = "nordic,pca10056-dk", "nordic,nrf52840-qiaa",
+		     "nordic,nrf52840";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,uart-mcumgr = &uart0;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led0: led_0 {
+			gpios = <&gpio0 13 GPIO_INT_ACTIVE_LOW>;
+			label = "Green LED 0";
+		};
+		led1: led_1 {
+			gpios = <&gpio0 15 GPIO_INT_ACTIVE_LOW>;
+			label = "Blue LED 1";
+		};
+		led2: led_2 {
+			gpios = <&gpio0 14 GPIO_INT_ACTIVE_LOW>;
+			label = "Red LED 2";
+		};
+	};
+
+	buttons {
+		compatible = "gpio-keys";
+		button0: button_0 {
+			gpios = <&gpio0 18 GPIO_PUD_PULL_UP>;
+			label = "Push button switch 0";
+		};
+	};
+
+	/* These aliases are provided for compatibility with samples */
+	aliases {
+		led0 = &led0;
+		led1 = &led1;
+		led2 = &led2;
+		sw0 = &button0;
+	};
+};
+
+&adc {
+	status ="ok";
+};
+
+&gpiote {
+	status ="ok";
+};
+
+&gpio0 {
+	status ="ok";
+};
+
+&gpio1 {
+	status ="ok";
+};
+
+&uart0 {
+	compatible = "nordic,nrf-uart";
+	current-speed = <115200>;
+	status = "ok";
+	tx-pin = <8>;
+	rx-pin = <7>;
+};
+
+&i2c0 {
+	status = "ok";
+	sda-pin = <5>;
+	scl-pin = <6>;
+};
+
```

Comment:
Maybe it would be good to add here:
```
&pwm0 {
	status = "ok";
	ch0-pin = <13>;
	ch0-inverted;
};
```
and enable the instance by default in `boards/arm/nrf52840_papyr/Kconfig.defconfig` to make it possible to build the basic samples "fade_led" and "blink_led" without providing dedicated `.conf` and `.overlay` files?